iTeChNoLoGiCi/lab1.asm

```diff
@@ -0,0 +1,26 @@
+model small
+.stack 100h
+.code
+
+start:
+    x1     db     3   
+    x2     db     1
+    x3     db     4
+    x4     db     1
+
+        mov al,x1

```

Comment:
Here  instruction - is ok, but indent before instruction = 8 spaces, it should be equal to 4.